master/master.cfg

```diff
@@ -723,32 +707,22 @@ create_builder('linux-32-gcc53', LLVM_TRUNK_BRANCH)
 create_builder('linux-64-gcc53', LLVM_TRUNK_BRANCH)
 create_builder('linux-32-gcc53', LLVM_8_BRANCH)
 create_builder('linux-32-gcc53', LLVM_9_BRANCH)
-create_builder('linux-64-gcc53', LLVM_8_BRANCH)
-create_builder('linux-64-gcc53', LLVM_9_BRANCH)
```

Comment:
Lots of linux and mac builders getting deleted here. Unintentional?

Reply:
yipes! no, wtf

Reply:
PTAL